chardet/charsetprober.py

```diff
@@ -113,14 +118,13 @@ def remove_xml_tags(buf):
         filtered = bytearray()
         in_tag = False
         prev = 0
-        buf = memoryview(buf).cast("c")
```

Comment:
The way this was before was added specifically as a performance improvement in #252. Calling `ord()` every time will slow this down a little.

Reply:
I see, thanks. This was done to workaround typeshed bug https://github.com/python/typeshed/issues/8182

I'll revert the change and then ignore the false positive.